fendermint/testing/graph-test/README.md

```diff
@@ -0,0 +1,20 @@
+# Integration test with The Graph
+
+## Prerequisite 
+0. Install `npm` if you haven't done so already.
```

Comment:
Maybe this can be added to the `fendermint/Makefile` as a requirement for `e2e` tests similar to how it warns about `protoc`

Reply:
Great suggestion! Fixed